### ISSUE TITLE
Update title of app according to page

### ISSUE THREAD

**Issue by Qrtn** [falling-fruit/falling-fruit-web#113]
maybe:
About | Falling Fruit
1991282 Blackberry | Falling Fruit

Ethan pointed out that titles of pages go into link previews in #636. Check how e.g. location pages look when sent via whatsapp!

- Use react-helmet

**Comment by wbazant** [comment]
The WhatsApp preview is currently based on meta description tag in index.html, and is:
```
A massive, collaborative map of the ... 
```

This issue won't help it, but we can have nicely labelled tabs on desktop, maybe so someone can bookmark a location.

**Comment by wbazant** [comment]
I always thought this would be a maintenance burden and lots of added work but with agents it wasn't. The titles I added are x | FallingFruit, with FallingFruit default. The default shows between pages when a component unloads, but will also catch any new routes where we forget a title.

I changed a few h3's or h2's that were page titles (user page, dataset page, confirm your email page) into h1's so we have fewer visual components doing the same thing.

**Comment by ezwelty** [comment]
> The WhatsApp preview is currently based on meta description tag in index.html, and is:
> 
> ```
> A massive, collaborative map of the ... 
> ```
> 
> This issue won't help it, but we can have nicely labelled tabs on desktop, maybe so someone can bookmark a location.

@wbazant That's true. Do we need a separate issue to dynamically set the description, photo, etc tags in index.html?

**Comment by wbazant** [comment]
It would need server side rendering! There's a possible setup with React where a page renders on the backend first and the HTML sent is a skeleton for client side rendering, where this benefit would maybe even come for free.
